src/providers/ModalsProvider.tsx

```diff
@@ -0,0 +1,63 @@
+import React, {
+  PropsWithChildren,
+  createContext,
+  useCallback,
+  useState,
+} from "react"
+import styles from "./ModalProvider.module.scss"
+
+interface ModalsContext {
+  content?: React.ReactNode
+  isOpen?: boolean
+  onPresent: (content: React.ReactNode, key?: string) => void
+  onClose: () => void
+}
+
+export const Context = createContext<ModalsContext>({
+  onPresent: () => undefined,
```

Comment:
`() => void` describes this action more precisely since technically you aren't returning undefined but rather nothing is happening, hence void. Also leverages TS here.

Reply:
Update: `() => {}`

Reply:
oh

Reply:
Just remembered. when I put the `()=>{}` for initializing the function, it gives me the eslint error.
![Screen Shot 2021-12-10 at 9 36 12 PM](https://user-images.githubusercontent.com/75422800/145624557-118d19c5-622e-42b4-a5a6-a5b19ad1f217.png)